lib/framework/procbot.ts

```diff
@@ -58,29 +59,52 @@ export class ProcBot {
 			return _.mapValues(value, ProcBot.injectEnvironmentVariables);
 		}
 		if (_.isString(value)) {
-			return ProcBot.stringInjector(value);
+			return ProcBot.stringInjector(value).cooked;
 		}
 		return value;
 	}
 
+	public static calculateInjections(value: any): Dictionary<string> {
```

Comment:
Don't like `calculateInjections` as a name as no actual transformation is occurring. `determineInjections`, perhaps?